dev-docs/bidders/loyal.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: bidder
+title: Loyal
+description: Prebid Loyal Bidder Adapter
+biddercode: loyal
+usp_supported: true
+gdpr_supported: true
+tcfeu_supported: true
```

Comment:
If you claim technical TCF support, you must have  gvl_id ( general vendor list ) and add it here